plugins/aws/rds/rdsTransportEncryption.js

```diff
@@ -0,0 +1,109 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'RDS Transport Encryption Enabled',
+    category: 'RDS',
+    description: 'Ensures RDS SQL Server instances have Transport Encryption enabled.',
+    more_info: 'Parameter group associated with the RDS instance should have transport encryption enabled to handle encryption and decryption',
+    link: 'https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL.html',
+    recommended_action: 'Update the parameter group associated with the RDS instance to have rds.force_ssl set to true',
+    apis: ['RDS:describeDBInstances', 'RDS:describeDBParameters', 'RDS:describeDBParameterGroups'],
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+        var regions = helpers.regions(settings);
+
+        async.each(regions.rds, function(region, rcb) {
+            var describeDBInstances = helpers.addSource(cache, source,
+                ['rds', 'describeDBInstances', region]);
+
+            if (!describeDBInstances) {
+                return rcb();
+            }
+
+            if (describeDBInstances.err || !describeDBInstances.data) {
+                helpers.addResult(results, 3,
+                    `Unable to query for RDS DB instances: ${helpers.addError(describeDBInstances)}`,
+                    region);
+                return rcb();
+            }
+
+            if (!describeDBInstances.data.length) {
+                helpers.addResult(results, 0, 'No RDS DB instances found');
+                return rcb();
+            }
+
+            var sqlInstanceFound = false;
+
+            async.each(describeDBInstances.data, function(db, cb){
+                if (db.Engine.startsWith('sqlserver')) {
+                    if (!db.DBInstanceArn) return cb();
+
+                    var resource = db.DBInstanceArn;
+                    sqlInstanceFound = true;
+
+                    if (!db.DBParameterGroups || !db.DBParameterGroups.length) {
+                        helpers.addResult(results, 0,
+                            `RDS DB instance "${db.DBInstanceIdentifier}" does not have any parameter groups associated`,
+                            region, resource);
+                    }
+
+                    var dbParameterGroup = db.DBParameterGroups[0].DBParameterGroupName;
```

Comment:
should we loop through db.DBParameterGroups

Reply:
I don't think so because:
 - You can only assign 1 parameter group with RDS database
 - There can't be more than 1 parameter groups because they can have conflicting properties